src/composables/Filter.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Indicate if text contains searched text.
+ * @param {String} textToFilter - Text to filter.
+ * @param {String} textToSearch - Text to search.
+ * @return {Boolean} True if `textToFilter` contains `textToSearch` otherwise false.
+ */
+export function textFilter(textToFilter, textToSearch) {
```

Comment:
textFilter -> searchFilter

Reply:
See with you, replace by `searchText`, done